hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -346,6 +356,8 @@ def setup_buildings_from_multiple_sources(
         if (
             any(
                 isinstance(geom, Polygon) for geom in self.exposure_geoms[0]["geometry"]
+            ) or any(isinstance(geom, MultiPolygon) for geom in self.exposure_geoms[0]["geometry"]
```

Comment:
Not sure if we should allow multipolygons to be honest. First of all good to check if FIAT allows for the use of multi-polygons in the first place (@dalmijn). 

I think probably a better approach would be to make sure we break all the provided Multi-polygons to standard polygons as a pre-processing step and log that change.